### ISSUE TITLE
Question about Logits

### ISSUE THREAD

**Issue by jisu0123** [markus93/NN_calibration#3]
Hello.
First of all, thank you for releasing a really good code. 
I tried to run your code using the logits you uploaded. 
However, your logit files can be downloaded via git-lfs because its size is very large. 
Without purchasing, I cannot download your logit files. 

Sorry to bother you, but could you upload the files to somewhere else like google drive?
I know this is so time-consuming..

Thanks !
